includes/collection/class-actors.php

```diff
@@ -76,6 +76,20 @@ public static function get_by_id( $user_id ) {
 	 * @return User|Blog|Application|WP_Error The Actor or WP_Error if user not found.
 	 */
 	public static function get_by_username( $username ) {
+		// Check if we're dealing with an old domain request.
+		$old_domain      = \get_option( 'activitypub_old_domain', '' );
+		$request_domain  = isset( $_SERVER['HTTP_HOST'] ) ? \sanitize_text_field( \wp_unslash( $_SERVER['HTTP_HOST'] ) ) : '';
+		$old_domain_host = ! empty( $old_domain ) ? \wp_parse_url( $old_domain, PHP_URL_HOST ) : '';
```

Comment:
Let's save it as just the host? Or do we need the full domain somewhere else?

Reply:
Yeah we should just need the host, will save some busywork